examples/json/server/main.go

```diff
@@ -46,7 +62,7 @@ func (h handler) Set(ctx context.Context, meta yarpc.Meta, req *SetRequest) (*Se
 
 func main() {
 	yarpc := yarpc.New(yarpc.Config{
-		Name: "myService",
+		Name: "keyvalue",
```

Comment:
Only local so far. I imagine we'll want the `RPC` to feed it to the inbounds and outbounds so that we have a name to advertise as and to send it in outgoing requests as the calling service's name.